mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -145,10 +148,18 @@ func VcrTest(t *testing.T, c resource.TestCase) {
 	if IsVcrEnabled() {
 		defer closeRecorder(t)
 	} else if isReleaseDiffEnabled() {
-		c = initializeReleaseDiffTest(c, t.Name())
+		// creates temporary file for the individual test, will be a temporary to store the output
+		tempOutputFile, err := createTemporaryFile()
+		if err != nil {
+			t.Errorf("creating temporary file %v", err)
+		}
+		defer func() {
+			writeOutputFileDeferFunction(tempOutputFile, t.Failed())
+		}()
+		c = initializeReleaseDiffTest(c, t.Name(), tempOutputFile)
 	}
 
-	c = extendWithTGCData(t, c)
+	//c = extendWithTGCData(t, c)
```

Comment:
I think this may need to be uncommented (also would presumably affect things outside of Diff Test)